src/agent0/core/hyperdrive/interactive/i_local_hyperdrive.py

```diff
@@ -162,8 +162,12 @@ class Config(IHyperdrive.Config):
         """The duration of a checkpoint (in seconds)."""
         curve_fee: FixedPoint = FixedPoint("0.01")  # 1%
         """The LP fee applied to the curve portion of a trade."""
-        flat_fee: FixedPoint = FixedPoint("0.0005")  # 0.05%
-        """The LP fee applied to the flat portion of a trade."""
+        # 0.05% APR. Here, we divide by 52 because the position duration is 1 week
+        # TODO do we want to default to 0.05% APR always? so we divide by position duration
```

Comment:
I'm asking Alex questions to clarify; will report back. I am pretty sure we do not want the user to have to know to divide by position duration, so it should not be divided here in the config. Maybe in `__post_init__`

Reply:
I'm torn on this. On one hand, I feel like it's good to reflect what's in solidity, but on the other, this fee in particular should perhaps be annualized so that the ux on this is sane. I do want to ensure that this matches the rest of our fees though